src/pricing-fetcher.ts

```
@@ -617,5 +640,48 @@
+
+				// Mock pricing data with context limits
+				const mockPricing = new Map([
+					['test-model', {
+						input_cost_per_token: 0.00001,
+						output_cost_per_token: 0.00003,
+						max_input_tokens: 100_000,
+					}],
+				]);
+				fetcher.cachedPricing = mockPricing;
+
+				const contextLimit = await Result.unwrap(fetcher.getModelContextLimit('test-model'));
```

ESLint

[error] Unsafe assignment of an error typed value.

(ts/no-unsafe-assignment)